skimage/exposure/_adapthist.py

```diff
@@ -70,7 +70,7 @@ def equalize_adapthist(image, ntiles_x=None, ntiles_y=None, clip_limit=0.01,
 
     References
     ----------
-    .. [1] http://tog.acm.org/resources/GraphicsGems/gems.html#gemsvi
+    .. [1] http://tog.acm.org/resources/GraphicsGems/
```

Comment:
I believe it is just a typo.
Also, there is one broken link in the header - https://github.com/ahojnnes/scikit-image/blob/7353785603eebe1f37bb98bdc44326e3e5f19707/skimage/exposure/_adapthist.py#L5 .